spyder/plugins/application/confpage.py

```diff
@@ -69,7 +69,8 @@ def setup_page(self):
                               'check_updates_on_startup')
 
         # Decide if it's possible to activate or not single instance mode
-        if running_in_mac_app():
+        # ??? Should this apply to all conda-based installers?
```

Comment:
```suggestion
```

I don't think it's necessary to require a single instance of our app on Windows and Linux. And perhaps we should remove the code below for our new Mac app because users have been requesting the ability to have multiple Spyder instances in that OS. But that's for another PR, I'd say.